plugin/path_login.go

```diff
@@ -216,10 +216,10 @@ func (b *GcpAuthBackend) getSigningKey(ctx context.Context, token *jwt.JSONWebTo
 		})
 		if err != nil {
 			// Attempt to get a normal Google Oauth cert in case of GCE inferrence.
-			key, err := gcputil.OAuth2RSAPublicKey(keyId, "")
-			if err != nil {
+			key, err2 := gcputil.OAuth2RSAPublicKey(keyId, "")
+			if err2 != nil {
 				return nil, errwrap.Wrapf(
-					fmt.Sprintf("could not find service account key or Google Oauth cert with given 'kid' id %s: {{err}}", keyId),
+					fmt.Sprintf("%s or could not find Google Oauth cert with given 'kid' id %s: {{err}}", err.Error(), keyId),
```

Comment:
Should one of these errors be err2? Also a very nit comment but I'd slightly prefer a variable name that isn't err2

Reply:
thank you for your comment.
I added a commit!